Makefile

```diff
@@ -39,3 +43,8 @@ test:
 
 types:
 	poetry run mypy questionary
+
+docs:
+	make -C docs html
+
+.PHONY: clean, install, formatter, lint, test, types, docs
```

Comment:
there is already a phony at the top, should we just add the additional ones there?

Reply:
I didn't notice that - I'll push a fix now!